whisk/api.go

```diff
@@ -71,6 +71,32 @@ type Api struct {
     GatewayFullPath string    `json:"gatewayFullPath,omitempty"`
     Swagger         string    `json:"swagger,omitempty"`
     Action          *ApiAction `json:"action,omitempty"`
+    PathParameters   []ApiPathParameter `json:"pathParameters,omitempty"`
+}
+
+type ApiPathParameter struct {
```

Comment:
since parameters can be included in more than just the path, i suggest renaming this struct to `ApiParameter`